src/App.js

```diff
@@ -1,29 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter } from 'react-router-dom';
 import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import NavBar from './components/NavBar';
+import AppRouter from './components/AppRouter';
+import Footer from './components/Footer';
```

Comment:
As you now know from Journeys the best is to change the order of the imports. I don't want to be obnoxious and don't think is necessary to comment more than once so this will be the only comment but look out for the other files